pyvolt-dpsim-demo/README.md

```diff
@@ -94,11 +100,11 @@ The configmap contains a demo dashboard and should automatically be recognized b
 
 ### CIM Editor Pintura
 
-The following installation will deploy a Pintura instance that is available at the nodePort specified in the pintura_values.yaml file. 
-Per default at port 31234: http://localhost:31234/
+The following installation will deploy a Pintura instance that is available at the nodePort specified in the pintura_values.yaml file.
+Per default at port 31234: http://localhost:31234/ (it may take a few moments to launch and become available)
```

Comment:
I thought it didn't work when I ran the command and immediately clicked on the link, but it just needed some time.